packages/desktop-client/src/components/reports/reports/Spending.tsx

```diff
@@ -224,13 +229,23 @@ export function Spending() {
                 >
                   Compare this month to:
```

Comment:
We also need to hide this when no data is present. Would probably be good to also do the `Spent MTD:` and `Spent Last MTD:`